Add make commands chatbot README (#281)
containers/ai-lab-recipes: recipes/natural_language_processing/chatbot/README.md

The [Podman Desktop](https://podman-desktop.io) [AI Lab Extension](https://github.com/containers/podman-desktop-extension-ai-lab) includes this recipe among others. To try it out, open `Recipes Catalog` -> `Chatbot` and follow the instructions to start the application.

# Build the Application

The rest of this document will explain how to build and run the application from the terminal, and will
go into greater detail on how each container in the Pod above is built, run, and 

> what purpose it serves in the overall application. A `Makefile` is available to further simplify building and running the application.

Reviewer: ```suggestion
what purpose it serves in the overall application. All the recipes use a central [Makefile](../../common/Makefile.common) that includes variables populated with default values to simplify getting started. Please review the [Makefile docs](../../common/README.md), to learn about further customizing your application.
```

> * [Makefile](#makefile)

Reviewer: ```suggestion
```

This application requires a model, a model service and an AI inferencing application.

* [Download a model](#download-a-model)
* [Build the Model Service](#build-the-model-service)
* [Deploy the Model Service](#deploy-the-model-service)
* [Build the AI Application](#build-the-ai-application)
* [Deploy the AI Application](#deploy-the-ai-application)
* [Interact with the AI Application](#interact-with-the-ai-application)
* [Embed the AI Application in a Bootable Container Image](#embed-the-ai-application-in-a-bootable-container-image)

> ## Makefile
> 
> Recipes use a central [Makefile](../../common/Makefile.common) that includes variables populated with default values.
> 
> ### Makefile variables
> 
> There are several [Makefile variables and targets](../../common/README.md) defined.
> It is recommended to edit the `Makefile` and change the variables or pass customized variables with `make` commands.
> 
> For example, from this directory:
> 
>   * `make APP_IMAGE=quay.io/your-repo/chatbot:latest build` will build the image from this [Containerfile](./app/Containerfile)
> 
>   * `make quadlet` can be used to configure the chatbot container along with a model-server and a model. This command
>   builds the application's metadata and generates Kubernetes YAML at `./build/chatbot.yaml` to spin up a Pod that can then be launched locally with:

Reviewer: ```suggestion
```
